src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/ILCompilerRIDs.props

```diff
@@ -6,6 +6,7 @@
     <OfficialBuildRID Include="linux-arm" Platform="arm" />
     <OfficialBuildRID Include="linux-musl-arm64" Platform="arm64" />
     <OfficialBuildRID Include="linux-arm64" Platform="arm64" />
+    <OfficialBuildRID Include="linux-riscv64" Platform="riscv64" />
```

Comment:
This gets included on the last line.

I think we can delete freebsd from this list as well. @dotnet/area-infrastructure does anyone remember if having community-platforms in this list necessary? See line 16 onwards. IIUC, freebsd later gets excluded in `src/installer/pkg/projects/Directory.Build.props` because it is not an official RID.

Reply:
This gets included on the last line.

I think we can delete freebsd from this list as well. @dotnet/runtime-infrastructure does anyone remember if having community-platforms in this list necessary? See line 16 onwards. IIUC, freebsd later gets excluded in `src/installer/pkg/projects/Directory.Build.props` because it is not an official RID.